source/includes/Encounter/encounter.md

```diff
@@ -198,7 +198,7 @@ fetch("/openmrs/ws/rest/v1/encounter?patient=96be32d2-9367-4d1d-a285-79a5e5db12b
 }
 ```
 
-* Quickly filter encounters with given query parameters. Returns a `404 Not Found` status if Encounter not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status returned.
+* Quickly filter encounters with given query parameters. Add `includeAll=true` parameter if wanting to include voided encounters. Returns a `404 Not Found` status if Encounter not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status returned.
```

Comment:
thanks @BartlomiejRasztabiga  we could actually say Add `includeAll=true` parameter if you want  to include voided encounters.

Reply:
Resolved here: https://github.com/openmrs/openmrs-contrib-rest-api-docs/pull/152/commits/5d5eaf25eda892e32a2ce92f9c33574aaea91440